lib/loader/index.d.ts

```diff
@@ -41,9 +44,9 @@ interface ASUtil {
   /** Reads the values of Uint32Array from the module's memory. */
   __getUint32Array(ptr: number): Uint32Array;
   /** Reads the values of Int32Array from the module's memory. */
-  __getInt64Array(ptr: number): BigInt64Array;
+  __getInt64Array?(ptr: number): BigInt64Array;
   /** Reads the values of Uint32Array from the module's memory. */
-  __getUint64Array(ptr: number): BigUint64Array;
+  __getUint64Array?(ptr: number): BigUint64Array;
```

Comment:
Wondering if this has implications on type checking, even though it's technically correct. An alternative might be to always create these functions, but make them throw if BigInt is not supported?

Reply:
> An alternative might be to always create these functions, but make them throw if BigInt is not supported?

In that case user should wrap call to try/catch or manually create check. Not sure this better way